api/C/include/csi_ll_pmp.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Low-Level PMP API
+ *
+ * This module provides functionality for PMP configuration.
+ *
+ * Copyright (c) RISC-V International 2023. Creative Commons License. Auto-
+ * generated file: DO NOT EDIT
+ */
+
+#ifndef CSI_LL_PMP_H
+#define CSI_LL_PMP_H
+
+#include "csi_types.h"
+
+/*
+ * Get PMP range number for the hard id. If the hart id is invalid, such as out of
+ * support range, it should return an error
+ *
+ * @param hart_id: The hart id used to get the pmp entry number
+ * @return : The number of PMP entry; -1 will be returned as and error code if the
+ * input is invalid.
+ */
+int csi_pmp_get_entry_num(unsigned int hart_id);
```

Comment:
Please change the name of this function to csi_pmp_get_num_entries, and change the text to say "number of PMP entries" rather than "range number" or "entry number", this is clearer.